3rdparty/bx/include/bx/mutex.h

```diff
@@ -31,6 +31,7 @@ namespace bx
 		void unlock();
 
 	private:
+		[[maybe_unused]]
```

Comment:
This is third-party code, so it would be better to turn off the warning for the "bx" project in scripts/src/3rdparty.lua (look for other uses of -Wno-unused) and that way the next pull from upstream won't overwrite it.